ui/src/adapters/api/connections.ts

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+import { z } from "zod";
+
+import { APIResponse, HTTPStatusSuccess, ResultOK, buildAPIError } from "src/adapters/api";
+import { Env } from "src/domain";
+import { buildAuthorizationHeader } from "src/utils/browser";
+import { API_VERSION, QUERY_SEARCH_PARAM } from "src/utils/constants";
+import { StrictSchema } from "src/utils/types";
+
+//TODO move it to credentials when is properly cleaned
+export interface Credential {
+  attributes: {
+    type: string;
+  };
+  id: string;
+}
+
+export interface Connection {
+  createdAt: string;
+  credentials: Credential[];
+  id: string;
+  issuerID: string;
+  userID: string;
+}
+
+export async function getConnections({
+  credentials = false,
```

Comment:
I'd set this to `true` by default and let the clients of this function decide when they want to save bandwidth.

Reply:
I'm going to leave it required to be aware of that field when is used

Reply:
Much better!